Sources/CocoaLumberjack/include/DDDispatchQueueLogFormatter.h

```diff
@@ -189,7 +191,4 @@ typedef NS_ENUM(NSUInteger, DDDispatchQueueLogFormatterMode){
 
 @end
 
-@interface DDAtomicCounter: NSObject<DDAtomicCountable>
```

Comment:
Oh, that one was public? 😬
Then we have to keep it but mark it as deprecated as well. Could have sworn it was only available internally...

Reply:
See 6ee355f